src/benchmark/file_io_read_micro_benchmark.hpp

```diff
@@ -0,0 +1,42 @@
+#include "micro_benchmark_basic_fixture.hpp"
+
+#include <fcntl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <numeric>
+
+namespace hyrise {
+
+const auto MB = uint32_t{1'000'000};
```

Comment:
Should be renamed to 'MEGA' based on other PR's feedback.

```suggestion
const auto MEGA = uint32_t{1'000'000};
```

Reply:
Will not be done to keep readibilty of 10 * MB instead of 10 * MEGA